packages/mobile/src/components/manager-mode-drawer/useShowManagerModeNotAvailable.tsx

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react'
+
+import { useDrawer } from 'app/hooks/useDrawer'
+
+import { useRoute } from 'app/hooks/useRoute'
+
+export const useShowManagerModeNotAvailable = () => {
+  const { params } = useRoute()
+  const { onOpen: openManagerModeDrawer } = useDrawer('ManagerMode')
+  useEffect(() => {
+    if ((params as { path: string })?.path === 'accounts-you-manage') {
+      openManagerModeDrawer()
+    }
+  }, [(params as { path: string })?.path])
```

Comment:
can we make a constant before useEffect and use that here? also i think `openMangagerModeDrawer` should be a dep?